sklearn/utils/__init__.py

```diff
@@ -3,36 +3,52 @@
 """
 import pkgutil
 import inspect
-from importlib import import_module
-from operator import itemgetter
-from collections.abc import Sequence
-from contextlib import contextmanager
-from itertools import compress
-from itertools import islice
+from itertools import (
+    compress,
+    islice,
+)
 import numbers
 import platform
 import struct
 import timeit
+import warnings
+
+from collections.abc import Sequence
+from contextlib import contextmanager
+from importlib import import_module
 from pathlib import Path
+from operator import itemgetter
 
-import warnings
 import numpy as np
 from scipy.sparse import issparse
 
-from .murmurhash import murmurhash3_32
-from .class_weight import compute_class_weight, compute_sample_weight
-from . import _joblib
+from .. import get_config
 from ..exceptions import DataConversionWarning
+
+from . import _joblib
+from .class_weight import (
+    compute_class_weight,
+    compute_sample_weight,
+)
 from .deprecation import deprecated
-from .fixes import np_version, parse_version
 from ._estimator_html_repr import estimator_html_repr
-from .validation import (as_float_array,
-                         assert_all_finite,
-                         check_random_state, column_or_1d, check_array,
-                         check_consistent_length, check_X_y, indexable,
-                         check_symmetric, check_scalar,
-                         _deprecate_positional_args)
-from .. import get_config
+from .fixes import np_version, parse_version
+from .multiclass import type_of_target
+from .murmurhash import murmurhash3_32
+from .validation import (
+    as_float_array,
+    assert_all_finite,
+    check_array,
+    check_consistent_length,
```

Comment:
I think for big PRs changing the import order can add too much noise.

For me, these changes make it harder for me to review and increases the chance for me to miss somethings.

It also makes the PR look bigger than it actually is.

Reply:
I died a little inside but I reverted those cosmetic changes :)